pytorch_lightning/strategies/strategy_registry.py

```diff
@@ -102,36 +102,36 @@ def get(self, name: str, default: Optional[Any] = None) -> Any:
         raise KeyError(err_msg.format(name, available_names))
 
     def remove(self, name: str) -> None:
-        """Removes the registered plugin by name."""
+        """Removes the registered strategy by name."""
         self.pop(name)
 
-    def available_plugins(self) -> List:
-        """Returns a list of registered plugins."""
+    def available_strategies(self) -> List:
+        """Returns a list of registered strategies."""
         return list(self.keys())
 
     def __str__(self) -> str:
-        return "Registered Plugins: {}".format(", ".join(self.keys()))
+        return "Registered Strategies: {}".format(", ".join(self.keys()))
 
 
-TrainingTypePluginsRegistry = _TrainingTypePluginsRegistry()
+StrategyRegistry = _StrategyRegistry()
 
 
-def is_register_plugins_overridden(plugin: type) -> bool:
+def is_register_strategies_overridden(strategy: type) -> bool:
```

Comment:
should we make these protected?

Reply:
I think we can have internal utility functions as unprotected.

Reply:
As users wouldn't have an use for it, we wouldn't have to worry about users using it.